faf-commons-data/src/main/java/com/faforever/commons/replay/header/Source.java

```diff
@@ -0,0 +1,9 @@
+package com.faforever.commons.replay.header;
+
+/**
+ * Populated by the engine, it represents the client identifiers and names of players that are connected to the game. The list includes observers.
+ * @param name
+ * @param sourceId
+ */
+public record Source(int sourceId, int PlayerId, String name) {
```

Comment:
playerId with proper casing

Reply:
Processed in:

- https://github.com/FAForever/faf-java-commons/pull/143/commits/07bee2553683c0662d68ea138f5f962097eca2fb